src/zookeeperLock.ts

```diff
@@ -166,15 +279,45 @@ export class ZookeeperLock extends EventEmitter {
         });
     };
 
+    private traceLog = () => {
+        setTimeout(() => {
+            const lifetime = Date.now() - this.created.getTime();
+            if (this.client) {
+                if (lifetime > 30000 && ((<any>this.client.getState()).name === 'SYNC_CONNECTED' || (<any>this.client.getState()).name === 'CONNECTED')) {
```

Comment:
30000, 30 seconds, just seemed like a random 'long time' to start squeaking about stuff, i can make it a config if you want